app/ui/lecture/taken-lecture/index.tsx

```diff
@@ -1,15 +1,29 @@
-import { fetchTakenLectures } from '@/app/business/services/lecture/taken-lecture.query';
+import { fetchTakenLectures, TakenLectureInfoResponse } from '@/app/business/services/lecture/taken-lecture.query';
 import TakenLectureList from './taken-lecture-list';
 import TakenLectureLabel from './taken-lecture-label';
 import TakenLectureAtomHydrator from '@/app/store/stores/taken-lecture-atom-hydrator';
 import UpdateTakenLecture from './update-taken-lecture';
 
+const processChapel = (takenLectures: TakenLectureInfoResponse[]) => {
+  return takenLectures.map((lecture) => {
+    return {
+      id: lecture.id,
+      year: lecture.year,
+      semester: lecture.semester,
+      lectureCode: lecture.lectureCode,
+      lectureName: lecture.lectureName,
+      credit: lecture.lectureName === '채플' ? 0.5 : lecture.credit,
+    };
+  });
+};
```

Comment:
credit 외 속성은 모두 그대로 사용하니, 구조 분해 할당을 사용하면 코드가 더 보기 편할 것 같아요.
```suggestion
const processChapel = (takenLectures: TakenLectureInfoResponse[]) => {
  return takenLectures.map((lecture) => {
    return {
      ...lecture,
      credit: lecture.lectureName === '채플' ? 0.5 : lecture.credit,
    };
  });
};
```

Reply:
알려주셔서 감사합니다
https://github.com/Myongji-Graduate/myongji-graduate-next/pull/139/commits/f0c4e6263b3af5236ee0c6486fbac5595cf1e937